3-nested-layouts.ipynb

```diff
@@ -665,7 +629,7 @@
     "\n",
     "Map type represents nested data where each value is a variable number of key-value pairs. Its physical representation is the same as a list of `{key, value}` structs.\n",
     "\n",
-    "The difference between a struct and a map type is that a struct holds the key in the schema therefore needs to be a string, the values are stored in values buffer, one per each field. There can be multiple keys and therefore multiple values buffers. The map comparing to the struct has one child array holding all different keys that need to be all same type but not necessarily string and second child array holding all the values which also need to be the same type.\n",
+    "The difference between a struct and a map type is that a struct holds the key in the schema therefore needs to be a string, the values are stored in the child arrays, one for each field. There can be multiple keys and therefore multiple child arrays. The map, on the other hand, has one child array holding all the different keys (that thus all need to be of the same type but not necessarily strings) and a second child array holding all the values, which also need to be of the same type.\n",
```

Comment:
The wording of `which also need to be of the same type` can be interpreted as needs to be of the same type as the keys.
```suggestion
    "The difference between a struct and a map type is that a struct holds the key in the schema therefore needs to be a string, the values are stored in the child arrays, one for each field. There can be multiple keys and therefore multiple child arrays. The map, on the other hand, has one child array holding all the different keys (that thus all need to be of the same type but not necessarily strings) and a second child array holding all the values, those values need to be of the same type (which doesn't have to match the one on the keys).\n",
```